tools/Microsoft.VisualStudio.Web.CodeGeneration.Design-arm/Microsoft.VisualStudio.Web.CodeGeneration.Design-arm.csproj

```diff
@@ -0,0 +1,6 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\Shared.proj" />
+  <PropertyGroup>
+    <RuntimeIdentifier>win-arm</RuntimeIdentifier>
```

Comment:
Have you tried using just one csproj instead? You can specified multiple RIDs in one file.
```
<RuntimeIdentifiers>win-arm;win-arm64;win7-x86</RuntimeIdentifiers>
```

Reply:
That's fine. Was just wondering if you were aware this was an option.